src/normal_form_game.jl

```diff
@@ -612,3 +612,50 @@ function pure2mixed(num_actions::Integer, action::PureAction)
     mixed_action[action] = 1
     return mixed_action
 end
+
+# is_pareto_efficient & is_pareto_dominant
+
+for args = ((:is_pareto_efficient, &), (:is_pareto_dominant, |))
```

Comment:
A variable name should be more descriptive here. For example use `f` for a function and maybe `op` for a logical operator?